sc2reader/resources.py

```diff
@@ -1021,8 +1017,6 @@ class GameSummary(Resource):
     that the data is not necessarily in the places we expect.
     """
 
-    url_template = "https://{}-s2-depot.classic.blizzard.com/{}.s2gs"
```

Comment:
Was this `url_template` unused, or am I not seeing the reference to it in the new code?

Reply:
This was unused.